### ISSUE TITLE
Value of passedChecks in policyReport always 0

### ISSUE THREAD

**Issue by bitsgalore** [veraPDF/veraPDF-apps#197]
it seems that the value of the *passedChecks*  attribute in the *policyReport* is always 0, even if this contradicts the information in the features report! 

For example, from the same call described [here](<https://github.com/veraPDF/veraPDF-apps/issues/196>), I end up with:

    <policyReport passedChecks="0" failedChecks="6" xmlns:vera="http://www.verapdf.org/MachineReadableReport">

But in this case the features report shows multiple occurrences of fonts that *are* embedded (and thus should have resulted in a passed check).

**Comment by BezrukovM** [comment]
Policy report bases on schematron validation. Each schematron rule can contain assertion and report elements. "Asserion" reports when its test fails. "Report" reports when its test passed. 
So:
failedChecks is a number of failed assertions.
passedChecks is a number of passed reports.

Our policy generation menu in the GUI lets you generate schematron file with assertions (without report elements). Hence if you will use policy files generated with our GUI menu you obtain passedChecks always equal to zero. 

But for people who needs more complex policy validation than provided by our GUI menu we have left that value because they may want to add some "report" rules into policy file and we will report their pass status.

**Comment by bitsgalore** [comment]
Yes that makes sense (incidentally I was using the CLI myself, wasn't even aware there was a policy generation menu in the GUI, good to know!).  Would suggest to clarify the meaning of failedChecks / passedChecks in the documentation, as this might confuse users.
